app/scss/_navigation.scss

```
@@ -9,7 +9,7 @@
     display: flex;
     align-items: center;
   }
-  &__item-right ul {
+  &__item-right .list {
```

Your solution isn't wrong. I'm just showing you a nested solution. The result will be the same as yours.

```suggestion
  &__item-right .list {
    display: flex;
    width: 100%;
    height: 100%;
    justify-content: space-evenly;
    list-style-type: none;
    color: white;
    font-family: "Open Sans", sans-serif;
    &__items {
      height: 100%;
      display: table;
      &--modify {
        color: white;
        text-decoration: none;
        height: 100%;
        display: inline-block;
        vertical-align: middle;
        display: table-cell;
        &:hover {
          color: #e58a20;
        }
      }
    }
  }
```